monai/inferers/merger.py

```diff
@@ -36,8 +36,18 @@ class Merger(ABC):
         device: the device where Merger tensors should reside.
     """
 
-    def __init__(self, output_shape: Sequence[int] | None = None, device: torch.device | str | None = None) -> None:
+    def __init__(
+        self,
+        output_shape: Sequence[int] | None = None,
+        crop_shape: Sequence[int] | None = None,
+        crop: bool = False,
```

Comment:
Thanks @KumoLiu for your comment. In PatchInferer the crop shape is calculated based on the actual size of the input image by default; however, I will revise it to make it clear for the user how things are working and move the cropping functionality to inferer instead of merger. Will let you know when it's done. Thanks